Assignment2Conditionals/src/assignment2conditionals/Assignment2Conditionals.java

```diff
@@ -0,0 +1,107 @@
+package assignment2conditionals;
+
+import java.util.Scanner;
+
+public class Assignment2Conditionals 
+{
+
+    public static void main(String[] args) 
+    {
+     Scanner scan = new Scanner (System.in);
+           
+     System.out.println("\t*************************************\n"
+                      + "\t*       Steve's Calculator          *\n"
+                      + "\t*************************************");
+          
+     System.out.println(" 1: Math\n 2: Subtraction\n 3: Multiplication"
+                     + "\n 4: Devision\n 5: Word Counter\n 6: Exit Program\n");
+     
+  
+     System.out.print("Please Pick An Option: ");
+     int userInputNum = scan.nextInt();
+     System.out.print("\n");
+      
+     
+     if(userInputNum == 1)
```

Comment:
Thank you! I considered using a switch.... about halfway through the program haha. So at that point I just decided to keep the else if statements.